lib/src/Base/Func/Evaluation.cxx

```diff
@@ -270,6 +270,19 @@ Graph Evaluation::draw(const Point & xMin,
   return getImplementation()->draw(xMin, xMax, pointNumber, scale);
 }
 
+/** Draw cross sections of the function assumed to have 1D output */
+GridLayout Evaluation::drawCrossCuts(const Point& centralPoint,
+  const Point& xMin,
```

Comment:
add a space before & here and in the other places

Reply:
done